Automattic/wp-calypso
# Make sure to show WPcom plans for atomic sites

### Changes proposed in this Pull Request

There's been instances in which owners of atomic sites were seeing Jetpack plans instead of WPcom plans when visiting the _Plans_ section. This PR fixes that.

Fixes 1164141197617539-as-1200980653703012

### Implementation notes

`isJetpackProductSite` returned `true` for Atomic sites whose `jetpack_connection_active_plugins` option had several items, which in turn caused `showJetpackPlans` to return `true` and display the Jetpack plans.

The fix consists in returning earlier in `isJetpackProductSite` for simple and Atomic sites.

Note that I transformed `is-jetpack-product-site.js` to `is-jetpack-product-site.ts`.

### Testing instructions

#### Prerequisites
Download the PR and run Calypso
Make sure you have a simple, atomic, and Jetpack self-hosted sites ready

#### Test
- For each site, visit `/plans/:site`
- Check that in the first 2 cases, WPcom plans are displayed, and in the last case, Jetpack plans are shown
- Check that the PR didn't introduce any error in the page or console
